scripts/ci/create-docker-image.sh

```diff
@@ -4,10 +4,13 @@ set -v
 set -x
 set -e
 
+[ "$WITH_SHA1" != true ] && WITH_SHA1=false
 # Arches are the docker arch names
 : ${ARCHES:=amd64 ppc64le s390x}
 # arm64 waiting on  golang 1.11 (https://github.com/skydive-project/skydive/pull/1188#discussion_r204336060)
-: ${DOCKER_IMAGE:=skydive/skydive}
+$WITH_SHA1 && \
```

Comment:
It seems you are either pushing to skydive/skydive-snapshots or skydive/skydive. We want both